pytorch_lightning/trainer/training_loop.py

```diff
@@ -736,9 +728,7 @@ def block_ddp_sync_behaviour(self):
         else:
             yield
 
-    def _process_closure_result(
-        self, batch_outputs: list, opt_idx: int
-    ) -> list:
+    def _process_closure_result(self, batch_outputs: list, opt_idx: int) -> list:
```

Comment:
```suggestion
    def _process_closure_result(self, batch_outputs: list, opt_idx: int) -> List:
```
you might need to import `List` from `typing` here.

Reply:
I think this change was made by the Black formatter. So `list` was being used beforehand as well.

Reply:
you can change it. no problem